npm/esbuild-wasm/bin/esbuild

```diff
@@ -41,12 +42,20 @@ process.on('exit', code => {
   // Otherwise if the exit code is zero, try to fall back to a binary N-API module that
   // calls the operating system's "exit(0)" function.
   const nativeModule = `${process.platform}-${os.arch()}-${os.endianness()}.node`;
-  const data = require('../exit0')[nativeModule];
-  if (data) {
+  const base64 = require('../exit0')[nativeModule];
+  if (base64) {
     try {
-      const tempFile = path.join(os.tmpdir(), `${Math.random().toString(36).slice(2)}-${nativeModule}`);
-      fs.writeFileSync(tempFile, zlib.inflateRawSync(Buffer.from(data, 'base64')));
-      require(tempFile);
+      const data = zlib.inflateRawSync(Buffer.from(base64, 'base64'));
+      const hash = crypto.createHash('sha256').update(data).digest().toString('hex').slice(0, 16);
```

Comment:
It would be faster to hash `base64` here rather than `data` due to its length:
```js
const hash = crypto.createHash('sha256').update(base64).digest().toString('hex').slice(0, 16);
```
What do you think?

Reply:
I made this change.